### DISCUSSION TITLE
when fedimint on testnet.... signet only isnt enough

### DISCUSSION THREAD

**Discussion by tuvok137** [fedimint/fedimint#2630] [General]
i would like to see a alpha version for testnet for better testing everything.
signet isnt enough.
but thx 4 the great work so far!!!

**Comment by tuvok137** [comment]
when nosrt github.
github should be just one hub and not the only hub.
bitcoin only yes
github only NO

**Comment by justinmoon** [comment]
You're free to deploy it to testnet whenever you like
